### PR TITLE
HAWQ-754. Open googletest build inside hawq for test usage.

### PR DESCRIPTION
**Pull request by xunzhang**

### CODE REVIEW

**Comment by xunzhang** [comment]
cc @yaoj2 @paul-guo-

**Comment by xunzhang** [comment]
In this pull request, I also fix comment bug for https://github.com/apache/incubator-hawq/pull/665.

**Comment by radarwave** [comment]
+1

File: GNUmakefile.in [hunk 1]
```diff
@@ -23,8 +22,7 @@ all:
 
 install:
 #	$(MAKE) -C doc $@
-# waiting for higher version of gcc ready
-#$(MAKE) -C depends/thirdparty/googletest install
+	$(MAKE) -C depends/thirdparty/googletest install
```
Comment (paul-guo-) [inline GNUmakefile.in]: Why not $@?

File: configure [hunk 2]
```diff
@@ -1556,7 +1556,7 @@ Optional Packages:
   --with-bonjour          build with Bonjour support
   --with-openssl          build with OpenSSL support
   --without-readline      do not use GNU Readline nor BSD Libedit for editing
-  --with-thrift       do not build thrift
+  --with-thrift       build thrift
```
Comment (paul-guo-) [inline configure]: ident issue? run "configure --h", I saw
  --without-readline      do not use GNU Readline nor BSD Libedit for editing
  --with-thrift       do not build thrift
  --without-libyarn       do not build libyarn

File: GNUmakefile.in (hunk 1 above)
Comment (xunzhang) [inline GNUmakefile.in]: They are the same, I have already modified.

File: configure (hunk 2 above)
Comment (xunzhang) [inline configure]: Fixed & Thanks!

**Comment by paul-guo-** [comment]
+1

**Pull request merged**